### PR TITLE
Case studies

### PR DESCRIPTION
**Pull request by erikrestificar**
Add Case Study feature to all pages

### CODE REVIEW

**Review by Jelena0921 — APPROVED** [review]

File: src/global-vars.scss [hunk 1]
```diff
@@ -50,3 +50,53 @@ $header-search-bar-height: 70px;
 $footer-height: 120px;
 $minimum-body-margin: 20px;
 $media-breakpoint: 600px;
+
+// Mobile
+$xSmallMaxScreenSize: 599px;
+
+// Tablet
+$smallMinScreenSize: 600px;
```
Comment (mimisun) [inline src/global-vars.scss:58]: remove unused values.

**Review by mimisun — CHANGES_REQUESTED** [review]

File: src/global-vars.scss [hunk 2]
```diff
@@ -50,3 +50,53 @@ $header-search-bar-height: 70px;
...
+ *     }
+ *
+ *     @include mobileMaxMediaQuery {
+ *        width: 100px; // Mobile
+ *     }
+ * ```
+ */
+@mixin laptopMaxMediaQuery {
+    @media only screen and (max-width: $mediumMaxScreenSize) {
+      @content;
```
[the remark this answers is not in the document]
Comment (mimisun) [reply src/global-vars.scss:83]: yes I know, but these queries are doing identical things which means they shouldn't be separate blocks?

File: src/global-vars.scss [hunk 3]
```diff
@@ -50,3 +50,48 @@ $header-search-bar-height: 70px;
...
+    }
+}
+  
+@mixin tabletMaxMediaQuery {
+    @media only screen and (max-width: $smallMaxScreenSize) {
+        @content;
+    }
+}
+
+@mixin mobileMaxMediaQuery {
```
Comment (mimisun) [inline src/global-vars.scss:93]: looks like this is the only media query that's being used, so you can delete the rest.
also we already have "$media-breakpoint: 600px" defined for mobile, so you could just use that variable here.

**Review by mimisun — APPROVED** [review]

File: src/global-vars.scss (hunk 3 above)
Comment (erikrestificar) [reply src/global-vars.scss:69]: done

**Pull request merged**